netjsonconfig/backends/openwrt/schema.py

```diff
@@ -928,6 +928,78 @@
                     },
                 },
             },
+            "zerotier": {
+                "type": "array",
+                "title": "ZeroTier Clients",
+                "uniqueItems": True,
+                "propertyOrder": 14,
+                "items": {
+                    "type": "object",
+                    "title": "ZeroTier Client",
+                    "additionalProperties": True,
+                    "required": ["id", "name"],
+                    "properties": {
+                        "id": {
+                            "type": "string",
+                            "propertyOrder": 1,
+                            "description": "The ID of the ZeroTier network",
+                        },
+                        "name": {
+                            "type": "string",
+                            "propertyOrder": 2,
+                            "description": "The name of the ZeroTier network",
+                        },
+                        "config_path": {
+                            "type": "string",
+                            "propertyOrder": 3,
+                            "description": (
+                                "The path to the persistent "
+                                "configuration folder (for ZT controller mode)"
+                            ),
+                        },
+                        "copy_config_path": {
+                            "type": "string",
+                            "propertyOrder": 4,
+                            "enum": ["0", "1"],
+                            "description": (
```

Comment:
Should this have a default?

Reply:
@atb00ker  According to the [sample configuration](https://github.com/mwarning/zerotier-openwrt/blob/master/zerotier/files/etc/config/zerotier) provided by the [OpenWrt ZeroTier package](https://github.com/mwarning/zerotier-openwrt), this property falls under advanced configuration and does not have a default value. Therefore, no default value is provided here. Please refer to the following resources for more information:

- [Sample Configuration File](https://github.com/mwarning/zerotier-openwrt/blob/master/zerotier/files/etc/config/zerotier): This file contains the necessary configurations for ZeroTier, including the `enabled` setting, a list of networks to join, and the `secret` (which can be automatically generated using [`zerotier-idtool` ](https://github.com/zerotier/ZeroTierOne/blob/dev/doc/zerotier-idtool.1.md) and is not required).
- [ZeroTier OpenWrt Wiki Page](https://github.com/mwarning/zerotier-openwrt/wiki#configure-zerotier-on-your-openwrt-router-using-uci): This wiki page provides guidance on configuring ZeroTier on your OpenWrt router using UCI.
- [init.d Script](https://github.com/mwarning/zerotier-openwrt/blob/master/zerotier/files/etc/init.d/zerotier): This script is responsible for managing the initialization of ZeroTier.